main_test.go

```diff
@@ -91,6 +98,7 @@ func TestPrune(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	cli.NegotiateAPIVersion(context.Background())
```

Comment:
I'm unsure if this is needed, but with this and the previous comment/snippet `client.FromEnv` I was successfully able to run the tests with Colima locally.